lib/gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -15,8 +15,8 @@ class LogfileConfiguration
 
     include ApplianceConsole::Logging
 
-    def initialize
-      self.disk                = nil
+    def initialize(config = {})
```

Comment:
This feels unnecessary. I would prefer setting disk using the `attr_accessor` rather than passing in an options hash.